polar-core/src/parser.rs

```diff
@@ -240,6 +240,22 @@ mod tests {
         assert_eq!(results[0].to_polar(), r#"a(x) if x = new Foo(1, 2);"#);
     }
 
+    #[test]
+    fn test_parse_new_mixed_args() {
+        let f = r#"a(x) if x = new Foo(1, 2, bar: 3);"#;
```

Comment:
Could we add a second kwarg here? I'd just feel better about having multiple positional + multiple kwargs in the test.